Overview.html

```diff
@@ -1923,6 +1923,32 @@ <h4 id="cors-protocol-and-credentials"><span class="secno">4.2.5 </span>CORS pro
   <td>Omitted
   <td>❌
```

Comment:
This doesn't appear display correctly in Chrome - it just shows up as a question mark...

![image](https://cloud.githubusercontent.com/assets/11558891/15023574/e7905896-11e5-11e6-9320-cb1922320cdb.png)